tests/test_post_process_results_format.py

```diff
@@ -1,17 +1,10 @@
 import unittest
 
-from pheval.post_processing.post_processing import (
-    PhEvalGeneResult,
-    PhEvalVariantResult,
-    RankedPhEvalGeneResult,
-    RankedPhEvalVariantResult,
-)
+from pheval.post_processing.post_processing import PhEvalGeneResult, PhEvalVariantResult
 
 from pheval_exomiser.post_process.post_process_results_format import (
     PhEvalGeneResultFromExomiserJsonCreator,
     PhEvalVariantResultFromExomiserJsonCreator,
-    create_pheval_gene_result_from_exomiser,
-    create_pheval_variant_result_from_exomiser,
```

Comment:
You ditched the entire test? Is there a test already for `create_standardised_results` which was not affected by the refactor?

Reply:
These tests were testing whether the correct data was extracted from the Exomiser json result and ranked/sorted correctly according to score. The ranking/sorting is all done in the methods in PhEval which have all the tests in the main package. So basically these tests were testing PhEval method functionality, which is already tested. There are tests in here for checking that the correct data is extracted from the Exomiser json result into the data class and then the final step of sorting and writing the outputs is tested in PhEval

Reply:
Ok :) Thanks for the clarification!